edge/conf/edge.yaml

```diff
@@ -33,6 +33,7 @@ edged:
     hostname-override: edge-node
     interface-name: eth0
     edged-memory-capacity-bytes: 7852396000
+    register-node: true
```

Comment:
now we are doing kubeedge component new api config feature, i suggest to add `register-node` field to new api struct.
https://github.com/kubeedge/kubeedge/blob/b61e49266715e493bbb5b2dc7593cc8c8289b8cc/pkg/apis/edgecore/v1alpha1/types.go#L103

Reply:
if  this `register-node` field is not configured,  is it automatically registered by default? I suggest to refer to kubelet。

Reply:
> i suggest to add register-node field to new api struct.

Agreed.

> if this register-node field is not configured, is it automatically registered by default? I suggest to refer to kubelet

Let's set the `register-node` default as `true`.

Reply:
done, PTAL